### PR TITLE
Render chart options

### PR DESCRIPTION
**Pull request by elayabharath**
+ Added chart options to toggle on / off data comparison & normalisation
+ Minor refactor to reuse the same grid across render functions

### CODE REVIEW

File: lib/view.dart [hunk 1]
```diff
@@ -104,13 +104,24 @@ void clearContentTab() {
   content.children.clear();
 }
 
-void renderAnalysisTabs(List<String> labels) {
-  var wrapper = html.DivElement()..classes = ['row'];
-  var labelCol = html.DivElement()
-    ..classes = ['col-lg-2', 'col-md-3', 'col-sm-12', 'col-xs-12']
-    ..innerText = 'Analyse';
-  var radioCol = html.DivElement()
+html.DivElement getGridRow() {
```
Comment (marianamarasoiu) [inline lib/view.dart]: nit: how about calling this `generateGridRowElement`? and `generateGridLabelColumnElement` and `generateGridOptionsColumnElement` below?

File: lib/controller.dart [hunk 2]
```diff
@@ -50,6 +56,11 @@ class AnalysisTabChangeData extends Data {
   AnalysisTabChangeData(this.tabIndex);
 }
 
+class ToggleOptionEnabledData extends Data {
+  bool isEnabled;
+  ToggleOptionEnabledData(this.isEnabled);
```
Comment (marianamarasoiu) [inline lib/controller.dart]: nit:
```suggestion
  bool enabled;
  ToggleOptionEnabledData(this.enabled);
```
`enabled` is fairly suggestive in itself that its type is bool. Same for the other `is*Enabled` vars in this file, e.g. `_isDataComparisonEnabled` -> `_dataComparisonEnabled`

**Review by marianamarasoiu — APPROVED** [review]
thanks @elayabharath , lgtm with a few nits!

File: lib/view.dart (hunk 1 above)
Comment (elayabharath) [reply lib/view.dart]: Yes this is much clearer, fixed

File: lib/controller.dart (hunk 2 above)
Comment (elayabharath) [reply lib/controller.dart]: ah yes makes sense, fixed

**Comment by elayabharath** [comment]
thanks @marianamarasoiu

**Pull request merged**